flink-runtime/src/test/java/org/apache/flink/runtime/deployment/ShuffleDescriptorTest.java

```diff
@@ -168,6 +168,7 @@ private static ResultPartitionDeploymentDescriptor createResultPartitionDeployme
 			STUB_CONNECTION_ID.getAddress().getPort());
 		PartitionDescriptor partitionDescriptor = new PartitionDescriptor(
 			new IntermediateDataSetID(),
+			1,
```

Comment:
pity, we still do not have a test builder for `PartitionDescriptor`

Reply:
you could cherry pick this hotfix commit with the `PartitionDescriptorBuilder`
https://github.com/azagrebin/flink/commit/342d05f658ac72259c5b797b40c753aad473a05e

Reply:
looking into it